bin/migrate-oats-data/menu/post_launch_commands/import_all.py

```diff
@@ -26,4 +26,7 @@ def import_all(console, args):
         console.log("Processing Documents")
         import_documents(batch_size=import_batch_size)
 
+        console.log("Process SRW Updates")
+        srw_survey_plan_update(batch_size=import_batch_size)
+
```

Comment:
Why is this separate from the general SRW import? Could you move it into the function that wraps all SRW-related ETL calls?

Reply:
Not possible, it needs to be done after the document import, because it depends on those file types being present. I could move it into the document import

Reply:
Thanks for clarifying, a comment highlighting the dependency will be good enough